ros2cli/ros2cli/node/direct.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rclpy
+
+# TODO(mikaelarguedas) revisit this once it's specified
+HIDDEN_NODE_PREFIX = '_'
+
+DEFAULT_TIMEOUT = 0.1
```

Comment:
I don't have any less arbitrary number in mind. 0.5, 1.0 seconds? I guess we can just wait and see. Someone should run for a while without the daemon to see what number may be most appropriate with a realistic system, like the turtlebot demo.